src/components/UserTimelineAPIContainer.jsx

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from "react";
+import TimelineContainer from "./TimelineContainer.jsx";
+import fetchUserTimeline from "../js/fetchUserTimeline.js";
+
+class UserTimelineAPIContainer extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            data: [],
+            fetchComplete: false,
+            errorOccurred: false
+        }
+
+        this.fetchData = () => {
+            fetchUserTimeline()
+            .then((data) => {
+                this.updateStatus(data);
+            })
+            .catch((error) => {
+                console.log(`Error occurred during UserTimelineContainer fetchData: ${error}`);
+                this.updateStatus("");
+            });
+        }
+
+        this.updateStatus = (responseData) => {
+            if(responseData === "") {
```

Comment:
be careful when using === "".  You are limiting what is not considered valid responseData